string value  # String value of the parameter
---
